# ra_velocity_controller:
#   type: "velocity_controllers/JointVelocityController"
#   joints:
#     - ra_shoulder_pan_joint
#     - ra_shoulder_lift_joint
#     - ra_elbow_joint
#     - ra_wrist_1_joint
#     - ra_wrist_2_joint
#     - ra_wrist_3_joint
#   constraints:
#       goal_time: 0.6
#       stopped_velocity_tolerance: 1.0
#       ra_shoulder_pan_joint: {trajectory: 0.3, goal: 0.1}
#       ra_shoulder_lift_joint: {trajectory: 0.3, goal: 0.1}
#       ra_elbow_joint: {trajectory: 0.3, goal: 0.1}
#       ra_wrist_1_joint: {trajectory: 0.3, goal: 0.1}
#       ra_wrist_2_joint: {trajectory: 0.3, goal: 0.1}
#       ra_wrist_3_joint: {trajectory: 0.3, goal: 0.1}
#   stop_trajectory_duration: 0.5
#   state_publish_rate:  25
#   action_monitor_rate: 10
#   allow_partial_joints_goal: true

--- 
gazebo_ros_control:
  pid_gains:
    ra_elbow_joint: 
      p: 70.0 
      i: 0.001 
      d: 1.0
    ra_shoulder_lift_joint: 
      p: 20.0 
      i: 0.01 
      d: 1.0
    ra_shoulder_pan_joint: 
      p: 20.0 
      i: 0.01 
      d: 1.0
    ra_wrist_1_joint: 
      p: 70.0 
      i: 0.001 
      d: 1.0
    ra_wrist_2_joint: 
      p: 20.0 
      i: 0.01 
      d: 1.0
    ra_wrist_3_joint: 
      p: 20.0 
      i: 0.01 
      d: 1.0
ra_elbow_motor: 
  joint: ra_elbow_joint
  pid: {p: 70, i: 0.01, d: 3.0}
  type: velocity_controllers/JointVelocityController
ra_shoulder_lift_motor: 
  joint: ra_shoulder_lift_joint
  pid: {p: 70, i: 0.01, d: 3.0}
  type: velocity_controllers/JointVelocityController
ra_shoulder_pan_motor: 
  joint: ra_shoulder_pan_joint
  pid: {p: 70, i: 0.01, d: 3.0}
  type: velocity_controllers/JointVelocityController
ra_wrist_1_motor: 
  joint: ra_wrist_1_joint
  pid: {p: 70, i: 0.01, d: 3.0}
  type: velocity_controllers/JointVelocityController
ra_wrist_2_motor: 
  joint: ra_wrist_2_joint
  pid: {p: 70, i: 0.01, d: 3.0}
  type: velocity_controllers/JointVelocityController
ra_wrist_3_motor: 
  joint: ra_wrist_3_joint
  pid: {p: 70, i: 0.01, d: 3.0}
  type: velocity_controllers/JointVelocityController

